libcst/matchers/_matcher_base.py

```diff
@@ -994,7 +994,7 @@ def _matches(
 def _construct_metadata_fetcher_null() -> Callable[
     [meta.ProviderT, libcst.CSTNode], object
 ]:
-    def _fetch() -> object:
+    def _fetch(*args: object, **kwargs: object) -> object:
```

Comment:
I apparently had no test for this and it failed when I tested it in `findall`. A regression for this is included in this PR.